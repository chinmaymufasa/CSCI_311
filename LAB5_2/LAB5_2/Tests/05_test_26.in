2
18
2
169
2
40
2
13
2
34
4
5
6
1
34
-1
