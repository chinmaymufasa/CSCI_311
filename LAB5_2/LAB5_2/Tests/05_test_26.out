40 18 13 34 169 
13 18 34 40 169 
13 34 18 169 40 
34
